include/oneapi/tbb/concurrent_queue.h

```diff
@@ -131,8 +133,43 @@ class concurrent_queue {
         queue_allocator_traits::destroy(my_allocator, my_queue_representation);
         r1::cache_aligned_deallocate(my_queue_representation);
     }
+  
+    concurrent_queue& operator=( const concurrent_queue& other ){
+      if (my_queue_representation != other.my_queue_representation) {
+	my_queue_representation = other.my_queue_representation;
```

Comment:
Such an implementation would not be correct. Since `my_queue_representation` is a pointer, we would just copy a pointer into another instance of queue. We would actually have two queues `other` and `*this` point to the same representation, not two independent queues with the same elements inside. Such an approach can be OK for move assignment, but in case of copy assignment, it can result in double-free than both queues would be destroyed.